package.json

```diff
@@ -41,6 +42,5 @@
     "scraper",
     "scraping"
   ],
-  "private": true,
```

Comment:
I haven't tested this project for npm imports, but yeah sure, removing this for future publishing. :+1: